src/cmd/main.go

```diff
@@ -17,6 +18,19 @@ func main() {
 
 	go setupHealthCheck()
 
+	setUpQueues()
+
+	// This loop keeps the app alive. This lets the pre-live deployment slot remain healthy
+	// even though it's configured without queues, which means we can quickly swap slots
+	// if needed without having to redeploy
+	for {
+		t := time.Now()
+		slog.Info(t.Format("2006-01-02T15:04:05Z07:00"))
```

Comment:
Do we want to log the time stamp every 10 minutes?  Will this fill out the logs unnecessarily?

Reply:
I figured it's a little nice to have something saying it's up and that every 10 minutes isn't very often, but we could make it less often or just skip the logging (there should at least be a health check running anyway). The thing that's _really_ filling up the logs is the `Trying to dequeue` message from `queue.go` 😅